candy-machine/js/.gitignore

```diff
@@ -1,2 +1,2 @@
 idl/*
-!idl/auction_house.json
\ No newline at end of file
+!idl/*.json
```

Comment:
We actually _do_ wanna check in the IDL so that other people can derive their SDKs from it if needed.